packages/checkbox/src/Checkbox.ts

```diff
@@ -116,24 +164,41 @@ export class Checkbox extends SizedMixin(CheckboxBase, {
         return html`
             ${super.render()}
             <span id="box">
-                ${checkmarkIcon[this.size as DefaultElementSize]}
-                ${dashIcon[this.size as DefaultElementSize]}
+                ${this.checked
```

Comment:
This is the DOM count win.

Rather than _always_ rendering both the checkmark AND dash icon, they'll now be left off the DOM _unless_ the element is `checked` or `indeterminate`.

A much more contained change than what you'd been looking at, but something that stays within the large "Spectrum" family until we're able to get the CSS techniques updated.

Reply:
Got it, that makes sense!